content/blog/2018/06/2018-06-26-lessons-java10-hackathon.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "What I learnt at the Jenkins & Java 10+ Hackathon"
```

Comment:
@tracymiranda 
Okay to say, "What I learned from the Jenkins Java 10+ Hackathon"?